.github/workflows/mvn-diff-nexus.yml

```
@@ -0,0 +1,46 @@
+on:
+  workflow_call:
+    inputs:
+      runner:
+        description: 'Runner for runs-on'
+        type: string
+        default: platform
+      path:
+        description: 'Mvn project folder'
+        required: false
+        default: '.'
+        type: string
+      target-branch:
+        description: 'Target branch to compare against'
+        required: true
+        type: string
+      java-version:
+        description: 'JDK version'
+        required: false
+        type: string
+        default: 17
+      java-distribution:
+        description: 'Distribution of the JDK'
+        default: 'temurin'
+        required: false
+        type: string
+      maven-version:
+        description: 'Maven version'
+        required: false
+        type: string
+        default: 3.9.0
+
+jobs:
+  mvn-diff-nexus:
+    uses: empathyco/platform-reusable-github-actions/.github/workflows/mvn-diff.yml@mvn-diff
```

Apply this change before merging
```suggestion
    uses: empathyco/platform-reusable-github-actions/.github/workflows/mvn-diff.yml@main
```